Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -186,6 +188,7 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
       return null;
     }
 
+    const {subviewStyle = {} } = this.props;
```

Comment:
You don't even have to a ternary operator (though I don't see any problems). `subviewStyle && subviewStyle[name]` works, and this is the pattern we use everywhere. For example we don't assign a default empty function when a callback is not given, we instead check if the function exists before executing it.